### ISSUE TITLE
Ability to rename attribute `data-id`.

### ISSUE THREAD

**Issue by StephanMeijer** [buttondown/tiptap-footnotes#11]
Currently in our application we use an `id` attribute on every element (except Text node).
We would like to reuse this name in your extension.
Is there any possibility you'd be willing to make the name of the `data-id`^[1]^[2] attributes configurable?

---

[^1]: https://github.com/buttondown/tiptap-footnotes/blob/main/package/src/footnotes/reference.ts#L43
[^2]: https://github.com/buttondown/tiptap-footnotes/blob/main/package/src/footnotes/footnote.ts#L30

**Comment by jmduke** [comment]
Yup, this seems pretty trivial to do. PR welcome, otherwise @mary-ext is out this week but I can entreat her to tackle it when she gets back!
